dist/ExtUtils-CBuilder/lib/ExtUtils/CBuilder/Platform/Windows/GCC.pm

```diff
@@ -95,8 +64,6 @@ sub format_linker_cmd {
     $spec{explib}             ,
```

Comment:
Just to catch this earlier, perhaps the test should make sure that the simplest possible compilation works instead of skipping with `skipped: no compiler available for testing`?